readme.md

```diff
@@ -179,7 +179,7 @@ You can customize the appearance of your `Stats Card` or `Repo Card` however you
 
 You can provide multiple comma-separated values in the bg_color option to render a gradient with the following format:
 
-    &bg_color=DEG,COLOR1,COLOR2,COLOR3...COLOR10
+&bg_color=DEG,COLOR1,COLOR2,COLOR3...COLOR10
```

Comment:
Please keep this indented or enclosed with a code block.

Reply:
My bad...seems my editor changed some formatting in the readme, I've enclosed it in a code block